templates/zerver/social_auth_select_email.html

```diff
@@ -57,8 +57,26 @@ <h1 class="get-started">Select account</h1>
         </form>
         {% endfor %}
     </div>
+    <p>
+        Only verified GitHub email addresses are listed above.
+    </p>
+    {% if unverified_emails %}
+    <p class="bottom-text">
+        Your GitHub account also has these unverified email addresses
+    </p>
+    <ul>
+        {% for email in unverified_emails %}
+        <li>
+            <p class="email">
+                {{ email }}
+            </p>
+        </li>
+        {% endfor %}
+    </ul>
     <p class="bottom-text">
-        Only verified GitHub email addresses are listed.
+        If you'd like to use one of these to login to Zulip via GitHub,<br>
+        you must <a href="https://github.com/settings/emails">verify it.</a>
     </p>
+    {% endif %}
```

Comment:
After playing with this design that shows the actual emails, it gives way too much weight to what is potentially basically useless data.  So I removed this, limiting it to just showing output if there are any.  That should be good enough for the use case; they can see the list in GitHub after clicking the link.